### ISSUE TITLE
Bad home page

### ISSUE THREAD

**Issue by z1haze** [yuliskov/SmartTubeNext#2823]
What happened to home, it's trash now. "All" is just full of old stale content. Why can't I have recently uploaded at the top of home?

Also the checkbox in settings to show old home view does absolutely nothing

**Comment by Helza** [comment]
Yeah I think YouTube changed something a couple months ago. It got pretty awful 😞 I think ppl even made videos about how bad it got all of a sudden.
